src/Sulu/Bundle/RouteBundle/Resources/config/doctrine/BaseRoute.orm.xml

```diff
@@ -19,6 +19,11 @@
                 <option name="default">1970-01-01 00:00:00</option>
             </options>
         </field>
+        <field name="changed" type="datetime" nullable="false">
```

Comment:
maybe there is something like `SET foreign_key_checks=0;` which disables this check? And writing an own SQL this way might be hacky, but I would prefer only the migration to be hacky than missing a `NOT NULL` validation in all future versions.